src/Ursa.Themes.Semi/Controls/TagInput.axaml

```diff
@@ -124,8 +124,8 @@
                     <DockPanel LastChildFill="True">
                         <PathIcon
                             Name="{x:Static u:ClosableTag.PART_CloseButton}"
-                            Width="8"
-                            Height="8"
+                            Theme="{StaticResource InnerPathIcon}"
+                            Classes="Small"
```

Comment:
Accept in this PR but subject to change. TagInput now have a default height 36, I need to update in the future and reduce it to 32.